tests/evaluation.at

```diff
@@ -118,3 +120,6 @@ POSITIVE_TEST([unpacking],[evaluation])
 POSITIVE_TEST([unsigned_operations], [evaluation])
 POSITIVE_TEST([unused_constraints],[evaluation])
 POSITIVE_TEST([x9],[evaluation])
+
+
+POSITIVE_FUNCTOR_TEST([functors],[evaluation])
```

Comment:
I know I'm the one that moved functors from interface to evaluation, but it *is* a form of interface test so perhaps it should be moved back ...

The important change was the addition of accepting the configuration flags.